### DISCUSSION TITLE
Eloquent destroy() ignore order

### DISCUSSION THREAD

**Discussion by ghost** [laravel/framework#33362] [General] [answered]
When I try to bulk delete models with destroy() the order of arguments/array is ignored.

For example Post::destroy(1, 2) and Post::destroy(2, 1) will always delete id 1 for first.

I haven't found a reference for that, it's normal behavior?

Laravel 7.11

**Comment by jmarcher** [comment, marked as answer]
The entries will be deleted in the natural order of your table.

```php
        foreach ($instance->whereIn($key, $ids)->get() as $model) {
            if ($model->delete()) {
                $count++;
            }
        }
```
That is the code used by `Model::destroy(...)`
